docs/devsite-help/protobuf.md

```diff
@@ -0,0 +1,142 @@
+# Protocol buffers in Google Cloud Libraries for .NET
+
+[Protocol buffers](https://developers.google.com/protocol-buffers) - also known as *protobuf* for short - are Google's language-neutral, platform-neutral, extensible mechanism for serializing structured data.
+
+Almost all the Google Cloud Libraries are generated from API
+definitions using protobuf, in the
+[GitHub googleapis/googleapis repository](https://github.com/googleapis/googleapis).
+
+Most protobuf messages in the APIs are simply generated with `protoc`
+(the Protocol Buffer Compiler) using the vanilla C# plugin. However,
+the generated partial classes are augmented in some libraries:
+
+- [Pagination](page-streaming.md) requires request/response types
+  to implement specific interfaces from GAX.
+- [Resource names](resource-names.md) add resource-name-typed properties
+  as strongly-typed "views" over string fields declared in the messages.
+- In some libraries, classes are augmented to provide simpler ways of
+  creating instances, such as via the `Filter` class in Datastore.
+
+For the most part, protobuf objects can be treated like normal mutable
+data transfer objects which happen to have well-defined binary and JSON
+formats. The sections below provide more information in areas which have
+previously caused some confusion.
+
+# Repeated fields and maps
```

Comment:
```suggestion
# Repeated and map fields
```

Reply:
Have gone with repeated fields and map fields - slightly clunkier, but I do want "repeated fields" together as a term.